lib/container/container.go

```diff
@@ -0,0 +1,93 @@
+// package container contains convenience functions for manipulating containers
+package container
+
+import (
+	"io"
+	"log"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+func RawStartContainer(opts docker.CreateContainerOptions) (string, error) {
+	client, err := docker.NewClient("unix:///var/run/docker.sock")
```

Comment:
Agreed on this being a problem. I think just using an environment variable would be a pretty good step in the right direction. (#71 setup to track this)